src/fieri/app/models/license_worker.rb

```diff
@@ -0,0 +1,43 @@
+require 'sidekiq'
+require 'net/http'
+
+class LicenseWorker
+  include ::Sidekiq::Worker
+
+  # Acceptable licenses were determined and documented in
+  # https://github.com/chef-cookbooks/cookbook-quality-metrics/blob/master/quality-metrics/qm-003-license.md
+  ACCEPTABLE_LICENSES = ['Apache 2.0', 'MIT', 'GNU Public License 2.0', 'GNU Public License 3.0'].freeze
```

Comment:
Help text from both `knife cookbook create` and `chef generate cookbook`:

```
-I LICENSE, --license LICENSE
Sets the license. Valid values are all_rights, apache2, mit, gplv2, or gplv3. Default value: all_rights.
```

License strings that appear in metadata from [`knife cookbook create`](https://github.com/chef/chef/blob/master/lib/chef/knife/cookbook_create.rb#L419-L430) match the list specified by QM003 and are as we have implemented the license check:

```
"Apache 2.0", "GNU Public License 2.0", "GNU Public License 3.0", "MIT"
```

License strings that appear in metadata from `chef generate cookbook`:

```
apache2 mit gplv2 gplv3
```

So I suggest at a minimum we add these short versions (as they appear in the help text and in `chef generate`'d metadata). Whether we move to using SPDX short identifiers is probably a bigger question that spans a number of ecosystem tools.